public/app/features/dashboard-scene/saving/provisioned/SaveProvisionedDashboard.tsx

```diff
@@ -111,6 +111,12 @@ export function SaveProvisionedDashboard({ drawer, changeInfo, dashboard }: Prop
   return (
     <form onSubmit={handleSubmit(doSave)}>
       <Stack direction="column" gap={2}>
+        {repositoryConfig?.readOnly && (
+          <Alert title="This repository is read only">
+            If you direct access to the target, copy the JSON and paste there
```

Comment:
```suggestion
            If you have direct access to the target, copy the JSON and paste it there.
```
Not sure what "there" means though.